lib/datadog/ruby_core_source/version.rb

```diff
@@ -0,0 +1,5 @@
+module Datadog
+  module RubyCoreSource
+    VERSION = '0.1.0'
+  end
+end
```

Comment:
I suggest keeping our version in sync with upstream -- this way it's really easy to know when we've fallen behind or not.

Reply:
But we will need some kind of a suffix in case we have multiple releases (e.g. to add a feature/fix a bug) for the same upstream version?